_pages/en_US/alternate-exploits.txt

```diff
@@ -6,6 +6,12 @@ title: "Alternate Exploits"
 
 3DS / 3DS XL / 2DS consoles on firmware 11.17.0 do not currently have any free, software-only methods. Therefore, all of the methods here have some sort of prerequisite.
 
+### Mario Kart 7
+
+A physical or digital (including preloaded) copy of Mario Kart 7 can be used in conjunction with Seedminer to install custom firmware.
+
+Continue to [Seedminer (Alternate)](seedminer-(alternate))
```

Comment:
Seedminer alternate is linked twice, first here and then right below this heading

If the intent is to use explicitly kartminer7 here, both headings should be merged